doc/extending/graph_rewriting.rst

```diff
@@ -400,10 +400,10 @@ varying number of arguments:
 >>> args_lv = var()
 >>> s = unify(cons(op_lv, args_lv), add(x, y))
 >>> s
-{~_2: <aesara.scalar.basic.Add at 0x7f54dfa5a350>, ~_3: e(x, y)}
+{~_2: <aesara.scalar.basic.Add object at 0x7fc277b841c0>, ~_3: ExpressionTuple((x, y))}
 >>> s = unify(cons(op_lv, args_lv), add(x, y, z))
 >>> s
-{~_2: <aesara.scalar.basic.Add at 0x7f54dfa5a350>, ~_3: e(x, y, z)}
+{~_2: <aesara.scalar.basic.Add object at 0x7fc277b841c0>, ~_3: ExpressionTuple((x, y, z))}
```

Comment:
Looks like we need to take another approach altogether here (e.g. one that doesn't print local locations/addresses).